src/analysis/normed/group/seminorm.lean

```diff
@@ -116,6 +157,12 @@ instance add_group_seminorm_class.to_zero_hom_class [add_group E] [add_group_sem
   zero_hom_class F E ℝ :=
 { ..‹add_group_seminorm_class F E› }
 
+@[priority 100] -- See note [lower instance priority]
+instance nonarch_add_group_seminorm_class.to_zero_hom_class [add_group E]
+  [nonarch_add_group_seminorm_class F E] :
+  zero_hom_class F E ℝ :=
+{ ..‹nonarch_add_group_seminorm_class F E› }
```

Comment:
```suggestion
```

Reply:
This one you can definitely remove, at the cost of calling `nonarch_add_group_seminorm_class.map_zero'` explicitly once, which is fine.